.github/workflows/labeler.yml

```diff
@@ -1,19 +1,13 @@
-name: Pull Request Labeler
-# This workflow is supposed to run every 5 minutes
+# https://github.com/marketplace/actions/labeler
+name: "Pull Request Labeler"
 on:
-  schedule:
-    - cron: '*/20 * * * *'
-permissions:
-  pull-requests:
-    write
+- pull_request_target
+
 jobs:
   triage:
     name: Update PR Labels
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
No reason at all, I thought latest was pointing to the same to be honest. I can change back :)

Reply:
Right I guess both are fine for now. Maybe `ubuntu-20.04` is more like a "pin" in case `ubuntu-latest` changes to another version later.

Reply:
>Maybe ubuntu-20.04 is more like a "pin" in case ubuntu-latest changes to another version later.

Indeed, but for these simple actions I think we will be fine with `latest`?

Not really running any tests on this workflow :)

Reply:
Sure. Looks like this was indeed related to some browser testing based on @bollwyvl comment: https://github.com/jupyterlab/jupyterlab/pull/9854#issuecomment-784648645

Reply:
Then if you agree, please approve 😄